forex-mtl/src/main/resources/application.conf

```diff
@@ -0,0 +1,7 @@
+app {
+  http {
+    host = "0.0.0.0"
+    port = 8080
+    timeout = 40 seconds
+  }
+}
```

Comment:
newline

Reply:
@abalonperin not sure what you mean with all these newline comments, can you elaborate?

Reply:
Nevermind, I guess this is the answer: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file